.github/workflows/publish.yml

```diff
@@ -13,17 +15,61 @@ jobs:
 
       - name: Set up Quarto
         uses: quarto-dev/quarto-actions/setup@v2
+        
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          pak-version: devel
+
+      - name: Render Quarto Project
+        uses: quarto-dev/quarto-actions/render@v2
+
+      - name: Restore lychee cache
+        uses: actions/cache@v3
         with:
-          # To install LaTeX to build PDF book
-          tinytex: true
-          # uncomment below and fill to pin a version
-          # version: SPECIFIC-QUARTO-VERSION-HERE
+          path: .lycheecache
+          key: cache-lychee-${{ github.sha }}
+          restore-keys: cache-lychee-
 
-      # add software dependencies here
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v1
+        with:
+          fail: true
+          args: "docs --exclude-loopback --insecure --exclude-mail --exclude-path docs/site_libs --cache --max-cache-age 1d"
 
-      - name: Publish to GitHub Pages (and render)
-        uses: quarto-dev/quarto-actions/publish@v2
+      - name: Read lychee's out.md
+        if: failure()
+        id: lychee-output
+        uses: juliangruber/read-file-action@v1
         with:
-          target: gh-pages
+          path: ./lychee/out.md
+
+      - uses: mshick/add-pr-comment@v2
+        if: failure()
+        with:
+          message: ${{ steps.lychee-output.outputs.content }}
+
+      - name: Save lychee cache
+        uses: actions/cache/save@v3
+        if: always()
+        with:
+          path: .lycheecache
+          key: ${{ steps.restore-cache.outputs.cache-primary-key }}
+
+      - name: Update cynkra org S3 bucket
+        if: contains(env.isExtPR, 'false')
+        timeout-minutes: 7
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # this secret is always available for github actions
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_KEY_CYNKRA_ORG }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_CYNKRA_ORG }}
+          AWS_DEFAULT_REGION: eu-central-1
+        run: |
+          curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+          unzip awscliv2.zip
+          sudo ./aws/install
+          aws configure list
+          # S3
+          aws s3 sync docs s3://cynkraweb-site
```

Comment:
This would push to the main website bucket. I don't think that we want to do that? Instead, we should push to a new bucket which is used as the source for a new Cloudfront instance.

Reply:
would you tweak this?